entity-framework/core/extensions/index.md

```diff
@@ -354,6 +354,12 @@ A source generator that can generate strongly-typed-id classes automatically for
 
 [GitHub repository](https://github.com/yangzhongke/LessCode.EFCore.StronglyTypedId)
 
+### Microsoft.EntityFrameworkCore.DynamicLinq
```

Comment:
Please list supported EF Core versions like it is done for all other extensions on this list.